src/Jenkinsfile

```diff
@@ -65,7 +65,7 @@ node {
         withCredentials([file(credentialsId: "jenkins-eccu-cache-purge", variable: 'EDGERC')]) {
           //path to .edgerc file is now set to $EDGERC"
           //Bust the current cache
-          sh "python3 bustCache.py $EDGERC ${APP_NAME}"
+          sh "python3 bustCache.py $EDGERC ${APP_NAME} ${BRANCH}"
```

Comment:
This did not work, I see OCM deploys running this without the new arg e.g.
https://jenkins-jenkins.5a9f.insights-dev.openshiftapps.com/job/insights-frontend-deployer/job/uhc-portal-frontend-deploy/job/prod-stable/30/console
I guess either `${BRANCH}` is blank, or old Jenkinsfile is still being used?

(not urgent, the rsync succeeds as before, only bustCache fails)

Reply:
@cben it did. You have a stale Jenkins file in your build repo. I've posted a bit more info into the Jira task. :slightly_smiling_face: